gulpfile.js

```diff
@@ -115,16 +115,39 @@ gulp.task('build/zone-node.js', ['compile-esm-node'], function(cb) {
 
 // Zone for the browser.
 gulp.task('build/zone.js', ['compile-esm'], function(cb) {
-  return generateScript('./lib/browser/rollup-main.ts', 'zone.js', false, cb);
+  return generateScript('./lib/browser/rollup-legacy-main.ts', 'zone.js', false, cb);
 });
 
 gulp.task('build/zone.min.js', ['compile-esm'], function(cb) {
-  return generateScript('./lib/browser/rollup-main.ts', 'zone.min.js', true, cb);
+  return generateScript('./lib/browser/rollup-legacy-main.ts', 'zone.min.js', true, cb);
+});
+
+// Zone for the evergreen browser.
+gulp.task('build/zone-evergreen.js', ['compile-esm'], function(cb) {
```

Comment:
if I'm not mistake, we still downlevel this to ES5. Could we ship it in ES2015 format instead? All evergreen browsers support it and this format will reduce the size of the bundle further. thanks!

Reply:
Yeah, you are right, I will do it.

Reply:
@IgorMinar, I have a question, because `uglifyjs` not really support `es6` (gulp-uglify can do, but it seems it doesn't support compression), should I use `babel+uglify` or use `closure` to generate the `zone-evergreen.min.js`, thanks!

Reply:
I think you need to use https://www.npmjs.com/package/terser to minify ES2015. Need some help?

I'd like to convert this repo to Bazel in prep for it moving into angular/angular anyway

Reply:
@alexeagle, thank you so much, sure, I definitely need your help on `terser`, and yeah, let's convert this to `Bazel`, that will be great!